tests/test_api.py

```diff
@@ -1,36 +1,20 @@
-import os
-import unittest
-from time import sleep
+from unittest import TestCase, main
 
-import requests
-from dotenv import load_dotenv
+from requests import get
```

Comment:
I wouldn't recommend this.
`requests.get` is much more clear than plain `get`

Reply:
`from requests import get as requests_get`? I prefer verbose imports and hate things like `import tkinter as tk` because its so unambiguous and I like knowing what I'm getting into the moment I open a file. Plus, I find it more readable.

Reply:
`from requests import get as requests_get` that's definitely better.

`import tkinter as tk` means you get all tkinter stuff, but separated under the `tk` namespace, and when you use an object it's immediately clear, what package it's from, without having to scroll up to the imports.
That said I better like to use plain `import tkinter`

Reply:
> `from requests import get as requests_get` that's definitely better.

Didn't work out for some of the modules. Are you sure it can work here?

Reply:
We only need to do this for requests, the rest are fine